projects/kubernetes/kubernetes/kubeadm-init.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/sh
 set -e
 touch /var/lib/kubeadm/.kubeadm-init.sh-started
-kubeadm init --skip-preflight-checks --kubernetes-version @KUBERNETES_VERSION@ $@
+if [ -f /etc/kubeadm/kubeadm.yaml ]; then
+  echo Using the configuration from /etc/kubeadm/kubeadm.yaml
```

Comment:
I wonder if at this point something like this might be appreciated by someone in the future:
```
if [ $# -ne 0 ] ; then
    echo WARNING: Ignoring command line options: $@
fi
```

Reply:
good idea. I'll add it.

Reply:
:+1: once you've pushed that give me a prod and I'll build the package so you can update `kube.yml`